src/styles/card.css

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__photo {
+  position: relative;
+  left: 3px;
+  width: 160px;
+  padding-bottom: 40px;
+}
+
+.card__title {
+  width: var(--card-width-inside);
+  font-size: 12px;
+  font-weight: 500;
+  padding-bottom: 4px;
+  color: #060b35;
+  line-height: 1.5;
+}
+
+.card__product-code {
+  width: var(--card-width-inside);
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;

```

Comment:
use variables for repeatable values

Reply:
take it into account, and other such cases

Reply:
didn't change it

Reply:
use variables for repeatable values